lib/repositories/movie/Impl/movie_from_json.dart

```diff
@@ -0,0 +1,35 @@
+import 'dart:convert';
+import 'package:flutter/services.dart';
+
+import '../i_movie_repository.dart';
+import '../../../data_classes/movie.dart';
+
+class MoviesFromJson extends MovieRepository {
+  static const moviesPath = 'jsonData/movies.json';
+  static late List<Movie> parsedMovies;
+
+  static Future<List<Movie>> parseMovies() async {
+    final jsonPath = await rootBundle.loadString('jsonData/movies.json');
```

Comment:
Same comment as before, you defined a static with the asset path, but here you continue using the direct string instead of using that new const

Reply:
done